bin/detail/toolchain_table.py

```diff
@@ -512,6 +512,7 @@ def verify(self):
       Toolchain('ios-nocodesign-11-3-dep-9-3-armv7', 'Xcode', ios_version='11.3', nocodesign=True),
       Toolchain('ios-nocodesign-11-3-dep-9-3-arm64', 'Xcode', ios_version='11.3', nocodesign=True),
       Toolchain('ios-nocodesign-11-3-dep-9-0-bitcode-cxx11', 'Xcode', ios_version='11.3', nocodesign=True),
+      Toolchain('ios-nocodesign-default-sdk-dep-9-0-cxx14', 'Xcode', nocodesign=True),
```

Comment:
Comparing with just `ios-nocodesign.cmake` I guess toolchain should be called just `ios-nocodesign-dep-9-0-cxx14.cmake`

Reply:
makes sense